sdk/data/azcosmos/cosmos_query_response.go

```diff
@@ -58,7 +58,7 @@ func newQueryResponse(resp *http.Response) (QueryItemsResponse, error) {
 }
 
 type queryServiceResponse struct {
-	Documents []map[string]interface{} `json:"Documents,omitempty"`
+	Documents []any `json:"Documents,omitempty"`
```

Comment:
In later versions of codegen, we've changed exposing raw formats like this to just `[]byte` so that callers can unmarshal the JSON into a structure of their own.  I don't know if that makes sense here, just pointing it out.

Also, since this is a breaking change, be sure to note that in the changelog.

Reply:
This is an internal type though, not public, how is this a breaking change ?

Reply:
Oh sorry I thought this was a return type exposed to callers.